rpcs3/Emu/RSX/VK/VKDMA.cpp

```diff
@@ -277,6 +277,10 @@ namespace vk
 		{
 			allow_host_buffers = true;
 		}
+#elif defined(__FreeBSD__)
```

Comment:
Looks similar to https://github.com/freebsd/freebsd-ports/commit/b8c1cb4ca5269441f4ba7d3828268f9099f702a9. However, gating on `__FreeBSD__` doesn't make sense. DRM_IOCTL_I915_GEM_USERPTR isn't implemented on any other BSD.

Reply:
Ah I did not know about other BSDs, FreeBSD was the only one I tried.